package.json

```diff
@@ -17,40 +17,44 @@
   "author": "Skroutz",
   "dependencies": {
     "bower": "~1.8.0",
-    "coffee-script": "^1.9.1",
-    "grunt": "^0.4.4",
-    "grunt-cli": "^0.1.13",
-    "grunt-contrib-coffee": "^0.13.0",
-    "grunt-contrib-compress": "^0.13.0",
-    "grunt-contrib-concat": "^0.5.1",
-    "grunt-contrib-copy": "^0.8.0",
-    "grunt-contrib-environment": "^0.1.1",
-    "grunt-contrib-uglify": "^0.8.0",
-    "grunt-hash": "^0.5.0",
-    "grunt-replace": "^0.8.0",
-    "grunt-shell": "^1.1.2",
-    "load-grunt-tasks": "^3.1.0",
-    "requirejs": "^2.1.16"
+    "coffee-script": "~1.12.4",
+    "grunt": "~1.0.1",
+    "grunt-cli": "~1.2.0",
+    "grunt-contrib-coffee": "~1.0.0",
+    "grunt-contrib-compress": "~1.4.1",
+    "grunt-contrib-concat": "~1.0.1",
+    "grunt-contrib-copy": "~1.0.0",
+    "grunt-contrib-environment": "~0.1.1",
+    "grunt-contrib-uglify": "~2.2.0",
+    "grunt-hash": "~0.5.0",
+    "grunt-replace": "~1.0.1",
+    "grunt-shell": "~2.1.0",
+    "load-grunt-tasks": "~3.5.2",
+    "requirejs": "~2.3.3"
   },
   "devDependencies": {
-    "grunt-bump": "0.3.0",
-    "grunt-contrib-clean": "^0.6.0",
-    "grunt-contrib-watch": "^0.6.1",
-    "grunt-karma": "^0.10.1",
-    "karma": "^0.12.31",
-    "karma-chai-sinon": "^0.1.4",
-    "karma-cli": "0.0.4",
-    "karma-coffee-preprocessor": "^0.2.1",
-    "karma-fixture": "^0.2.2",
-    "karma-html2js-preprocessor": "^0.1.0",
-    "karma-mocha": "^0.1.10",
-    "karma-mocha-reporter": "^1.0.0",
-    "karma-phantomjs-launcher": "^0.1.4",
-    "karma-requirejs": "^0.2.2",
-    "karma-coverage": "^0.2.7",
-    "ibrik": "Constellation/ibrik",
-    "istanbul": "^0.3.7",
-    "mocha": "^2.1.0",
-    "supertest": "^0.15.0"
+    "grunt-bump": "~0.8.0",
+    "grunt-contrib-clean": "~1.0.0",
+    "grunt-contrib-watch": "~1.0.0",
+    "grunt-karma": "~1.0.0",
+    "karma": "~1.5.0",
+    "karma-chai-sinon": "~0.1.5",
```

Comment:
Consider using `^0.1.5`, see [this comment](https://github.com/skroutz/analytics.js/pull/31/commits/732897e18e3d86bb82806f298d569a3698bd907a#r108881075).